### PR TITLE
add example to insert on conflict

### LINKED ISSUE #2378: Example insert on conflict referencing values from table
Add another example adding to the existing value in the row.

https://www.cockroachlabs.com/docs/stable/insert.html#update-values-on-conflict

```
SELECT * FROM accounts WHERE id = 8;

+----+---------+
| id | balance |
+----+---------+
|  8 |   500.5 |
+----+---------+

INSERT INTO accounts (id, balance)
    VALUES (8, 500.50)
    ON CONFLICT (id)
    DO UPDATE SET balance = accounts.balance + excluded.balance;

SELECT * FROM accounts WHERE id = 8;

+----+---------+
| id | balance |
+----+---------+
|  8 |   1001 |
+----+---------+
```

### PR DESCRIPTION
**Pull request by m-schneider**
Closes #2378

### CODE REVIEW

File: v1.1/insert.md [hunk 1]
```diff
@@ -44,6 +44,15 @@ Parameter | Description
 
 ## Examples
 
+First let's create a table accounts.
+
+~~~ sql
+> CREATE TABLE accounts(
+    id INT NOT NULL UNIQUE,
```
Comment (jseldess) [inline v1.1/insert.md]: The `SHOW` output below shows the `id` column using `unique_rowid()` as the default value. Can you add that to this `CREATE TABLE`?

File: v2.0/insert.md [hunk 2]
```diff
@@ -44,6 +45,15 @@ Parameter | Description
 
 ## Examples
 
+First let's create a table accounts.
+
+~~~ sql
+> CREATE TABLE accounts(
+    id INT NOT NULL UNIQUE,
```
Comment (jseldess) [inline v2.0/insert.md]: The `SHOW` output below shows the `id` column using `unique_rowid()` as the default value. Can you add that to this `CREATE TABLE`?

File: v1.1/insert.md [hunk 3]
```diff
@@ -44,6 +44,15 @@ Parameter | Description
 
 ## Examples
 
+First let's create a table accounts.
```
Comment (jseldess) [inline v1.1/insert.md]: Let's change this intro a little: 

> All of the examples below assume you've already created a table `accounts`:

File: v2.0/insert.md [hunk 4]
```diff
@@ -44,6 +45,15 @@ Parameter | Description
 
 ## Examples
 
+First let's create a table accounts.
```
Comment (jseldess) [inline v2.0/insert.md]: Let's change this intro a little: 

> All of the examples below assume you've already created a table `accounts`:

**Comment by jseldess** [comment]
I can make the final edits, @m-schneider.

**Comment by m-schneider** [comment]
I've updated the files.

---

Review status: 0 of 2 files reviewed at latest revision, 4 unresolved discussions.

---

*[v1.1/insert.md, line 47 at r1](https://beta.reviewable.io/reviews/cockroachdb/docs/2666#-L76LgQxgMGFTphD88Dy-r1-47:-L7QhLbo-uNUyPvBqoMY:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/d13ac2692b73dc7dce828b74ab78e9150c3d9736/v1.1/insert.md#L47)):*
<details><summary><i>Previously, jseldess wrote…</i></summary><blockquote>

Let's change this intro a little: 

> All of the examples below assume you've already created a table `accounts`:

</blockquote></details>

Done.

---

*[v1.1/insert.md, line 51 at r1](https://beta.reviewable.io/reviews/cockroachdb/docs/2666#-L76LgQxgMGFTphD88Dy-r1-51:-L7QhLCg9qsI8UHrF5Hv:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/d13ac2692b73dc7dce828b74ab78e9150c3d9736/v1.1/insert.md#L51)):*
<details><summary><i>Previously, jseldess wrote…</i></summary><blockquote>

The `SHOW` output below shows the `id` column using `unique_rowid()` as the default value. Can you add that to this `CREATE TABLE`?
</blockquote></details>

Done.

---

*[v2.0/insert.md, line 48 at r1](https://beta.reviewable.io/reviews/cockroachdb/docs/2666#-L76LgQxgMGFTphD88Dz-r1-48:-L7QhOaTC8tpJ6GnBar2:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/d13ac2692b73dc7dce828b74ab78e9150c3d9736/v2.0/insert.md#L48)):*
<details><summary><i>Previously, jseldess wrote…</i></summary><blockquote>

Let's change this intro a little: 

> All of the examples below assume you've already created a table `accounts`:

</blockquote></details>

Done.

---

*[v2.0/insert.md, line 52 at r1](https://beta.reviewable.io/reviews/cockroachdb/docs/2666#-L76LgQxgMGFTphD88Dz-r1-52:-L7QhOS1BdeyUqhn-Orv:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/d13ac2692b73dc7dce828b74ab78e9150c3d9736/v2.0/insert.md#L52)):*
<details><summary><i>Previously, jseldess wrote…</i></summary><blockquote>

The `SHOW` output below shows the `id` column using `unique_rowid()` as the default value. Can you add that to this `CREATE TABLE`?
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://beta.reviewable.io/reviews/cockroachdb/docs/2666#-:-L7QhP7_4-UNsCkvR7pL:bcqrba1)*

File: v1.1/insert.md [hunk 5]
```diff
@@ -44,6 +44,15 @@ Parameter | Description
 
 ## Examples
 
+All of the examples below assume you've already created a table accounts:
```
Comment (jseldess) [inline v1.1/insert.md]: nit: wrap `accounts` in backticks.

Comment (jseldess) [reply v1.1/insert.md]: Also in other versions of the page.

File: v2.0/insert.md [hunk 6]
```diff
@@ -11,7 +11,8 @@ The `INSERT` [statement](sql-statements.html) inserts one or more rows into a ta
 ## Performance Best Practices
 
 - A single [multi-row `INSERT`](#insert-multiple-rows-into-an-existing-table) statement is faster than multiple single-row `INSERT` statements. To bulk-insert data into an existing table, use a multi-row `INSERT` instead of multiple single-row `INSERT` statements.
-- The experimental <a href=import.html><code>IMPORT</code></a> statement performs better than <code>INSERT</code> when inserting rows into a new table. 
+- The experimental <a href=import.html><code>IMPORT</code></a> statement perfo
```
Comment (jseldess) [inline v2.0/insert.md]: Not sure why the line is wrapping mid-word. While you're editing this page, can you fix that. Also, it'd be great if you could replace the html for Markdown:

```
The experimental [`IMPORT`](import.html) statement performs better than `INSERT` when inserting into a new table.
```

**Comment by m-schneider** [comment]
Review status: 0 of 2 files reviewed at latest revision, 6 unresolved discussions, some commit checks pending.

---

*[v1.1/insert.md, line 47 at r2](https://reviewable.io:443/reviews/cockroachdb/docs/2666#-L76LgQxgMGFTphD88Dy-r2-47:-L7QlcRMQwdBBf5PwECf:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/4c371d6db0071d0f69d25e16a72643ddff2bc379/v1.1/insert.md#L47)):*
<details><summary><i>Previously, jseldess wrote…</i></summary><blockquote>

Also in other versions of the page.
</blockquote></details>

Done.

---

*[v2.0/insert.md, line 14 at r2](https://reviewable.io:443/reviews/cockroachdb/docs/2666#-L76LgQxgMGFTphD88Dz-r2-14:-L7QmVsmnrf5yI_rW2FD:b-qnxz3l) ([raw file](https://github.com/cockroachdb/docs/blob/4c371d6db0071d0f69d25e16a72643ddff2bc379/v2.0/insert.md#L14)):*
<details><summary><i>Previously, jseldess wrote…</i></summary><blockquote>

Not sure why the line is wrapping mid-word. While you're editing this page, can you fix that. Also, it'd be great if you could replace the html for Markdown:

```
The experimental [`IMPORT`](import.html) statement performs better than `INSERT` when inserting into a new table.
```
</blockquote></details>

I also changed it down below.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/docs/2666)*

**Comment by jseldess** [comment]
Thanks, @m-schneider.

**Pull request merged**